frontend/packages/console-shared/src/components/editor/YAMLEditor.scss

```diff
@@ -1,13 +1,11 @@
 .ocs-yaml-editor {
   &__root {
     flex: 1;
-    overflow: hidden;
```

Comment:
@rohitkrai03 do you recall why the `overflow: hidden` was present?

Reply:
I think it was present to make sure that the YAML editor would adapt properly for any screen size but in retrospect I guess it's not necessary to have `overflow: hidden` on `.ocs-yaml-editor__root` because it's already there on `.ocs-yaml-editor__wrapper`.

Reply:
I tested it just now with `overflow: hidden` removed and it's working fine. Can't think of a scenario where it seems to break anything.